README.md

```diff
@@ -1,2 +1,83 @@
 # scm-github-graphql
-Module to query github graphql api
+[![Version][npm-image]][npm-url] ![Downloads][downloads-image] [![Build Status][status-image]][status-url] [![Open Issues][issues-image]][issues-url] ![License][license-image]
+
+## Usage
+
+```bash
+npm install screwdriver-scm-github-graphql
+```
+
+### Initialization
+
+The class interacts with (GitHub GraphQL API)[https://docs.github.com/en/enterprise-cloud@latest/graphql/overview/about-the-graphql-api]
+The following configuration is needed
+
+| Parameter        | Type  |  Default | Description |
+| :-------------   | :---- | :-------------| :-------------|
+| config        | Object | | Configuration Object |
+| config.graphQlUrl | String | https://api.github.com/graphql | Github GraphQL API Endpoint |
+```js
+const scm = new GithubScmGraphQL({
+    graphQlUrl: 'https://api.github.com/graphql'
+});
+```
+
+### Methods
+
+#### getEnterpriseUserAccount
+Required parameters:
+
+| Parameter        | Type  |  Description |
+| :-------------   | :---- | :-------------|
+| schema     | Object | The schema object |
+| schema.slug     | String | The github enterprise slug |
+| schema.username      | String | The github user's login name |
```

Comment:
is there a particular reason why this one is `username` vs `login`?

Reply:
`login` is used in GQl context, I guess I can change this to `login`